external/stdk/Kconfig

```diff
@@ -15,6 +15,12 @@ config STDK_IOT_CORE
 
 if STDK_IOT_CORE
 
+config STDK_CURRENT_TIME_YEAR
+	int "Current year setting"
+	default 2019
+	---help---
+		Current year used to check if system time is updated by ntp client.
```

Comment:
this patch just work for now or for future?

Reply:
this value could work even for future, but as discussed above, this kconfig also seems weired, I will remove this , and apply different solution for time delay